br/br-snapshot-manual.md

```diff
@@ -287,3 +287,46 @@ tiup br restore full\
     --crypter.method aes128-ctr \
     --crypter.key 0123456789abcdef0123456789abcdef
 ```
+
+## Checksum
+
+Checksum is a method used by the BR tool to verify the integrity of backup and restore data. BR supports two levels of checksums:
+
+1. **File-level checksum**: Verifies the backup files themselves to ensure integrity during storage and transmission. This checksum is always enabled and cannot be disabled.
+2. **Table-level checksum**: Verifies the integrity of table data content and confirms the business logic consistency of the data. This checksum is disabled by default but can be enabled through parameters.
```

Comment:
```suggestion
2. **Table-level checksum**: Verifies the integrity of table data content and confirms the business logic consistency of the data. You can enable or disable this checksum through parameters.
```

Reply:
hmm, I feel like it will not be clear to customer that we disable it by default and they might think it's still enabled?